google/ggrc-core
# QUICK-FIX Cleanup query builder

# Issue description

This is just prep work for custom api endpoints for object history and related assessments. 
Since bothe of those will need pagination, we should ensure that all places will use the same functions for order by and limit.

# Steps to test the changes

Check that order by and limit still work. The order by should be tested with more than a single column like clicking on the top of tree view supports.

# Solution description

I have encapsulated and separated limit and order by functions and moved them to their own module. 

# Sanity checklist

- [x] My changes are covered by tests. - even though there are no tests in this PR I have verified that the changes here are covered by inducing small bugs and romeving order by clauses.



# PR Review checklist
